src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -22,6 +22,7 @@
 import java.util.WeakHashMap;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.atomic.AtomicLong;
```

Comment:
I know you told us to ignore the linting failures, but: why is this import here?